thread/thread.cpp

```diff
@@ -86,7 +86,7 @@ inline int posix_memalign(void** memptr, size_t alignment, size_t size) {
                            #name": "
 #endif
 
-static constexpr size_t PAGE_SIZE = 1 << 12;
+static const size_t PAGE_SIZE = getpagesize();
```

Comment:
Turns out our ARM64 testing runner built on a kernel with 64KB page size, that makes all tests and codes that assumed pagesize is 4KB failed.